lib/modules/platform/github/massage-markdown-links.spec.ts

```diff
@@ -87,4 +87,16 @@ describe('modules/platform/github/massage-markdown-links', () => {
       expect(massageMarkdownLinks(input)).toEqual(output);
     }
   );
+
+  it('replace url contains disappear text inline (colors)', () => {
+    const input = [
+      'pnpm rebuild accepts --store-dir by @&#8203;user in https://github.com/foo/bar/issues/1',
+    ].join('\n');
+    const res = massageMarkdownLinks(input);
+    expect(res).toEqual(
+      [
+        'pnpm rebuild accepts --store-dir by @user in [https://github.com/foo/bar/issues/1](https://togithub.com/foo/bar/issues/1)',
```

Comment:
Your **after fix** PR is pinging _all_ the authors mentioned in the release notes. 🙈

I suspect you can fix this by putting the `@user` parts between backticks or a `<code></code>` block, like this:

```ts
        'pnpm rebuild accepts --store-dir by `@user` in [https://github.com/foo/bar/issues/1](https://togithub.com/foo/bar/issues/1)',
```

**Bad (current behavior):**

> blabla by @HonkingGoose in [link to PR](#)

**Correct:**

> blabla by `@HonkingGoose` in [link to PR](#)

Reply:
I added backticks around @user but there is no link to the user profile, is it the correct behavior?
**result:**
https://github.com/MaronHatoum/16921/pull/45

Reply:
Looks like there is still one wrong username
![image](https://user-images.githubusercontent.com/1798109/188559455-eee948db-9c36-4129-ac93-ffed08dc848a.png)

Reply:
can you check now please:
https://github.com/MaronHatoum/16921/pull/46